cmd/kube-apiserver/app/options/validation.go

```diff
@@ -21,7 +21,7 @@ import (
 	"fmt"
 
 	apiextensionsapiserver "k8s.io/apiextensions-apiserver/pkg/apiserver"
-	utilfeature "k8s.io/apiserver/pkg/util/feature"
+	utilfeature "k8s.io/component-base/pkg/feature"
```

Comment:
Would `featuregate "k8s.io/component-base/pkg/feature"` be better? Didn't change the import alias to reduce # of changes in this PR but I can do this if it makes sense.

Reply:
Let's keep the changes in this PR to mostly mechanical. You can followup with another tiny PR on that.

Reply:
works for me, though I don't think the follow-up will be tiny ;)